charts/tileserver/templates/service.yaml

```
@@ -0,0 +1,28 @@
+{{- with .Values.api.service }}
+---
+apiVersion: v1
+kind: Service
+
+metadata:
+  name: {{ include "tileserver.fullname" $ }}
+  {{- if .annotations }}
+  annotations:
+    {{- toYaml .annotations | nindent 4 }}
+  {{- end }}
+  labels:
+    {{- include "tileserver.labels" $ | nindent 4 }}
+    {{- if .labels }}
+    {{- toYaml .labels | nindent 4 }}
+    {{- end }}
+
+spec:
+  type: {{ .type }}
+  ports:
+    - port: {{ .port }}
+      name: http
+      targetPort: http
+
```

Я бы еще добавил возможность поменять `externalTrafficPolicy`.
```suggestion

  {{- if .externalTrafficPolicy }}
  externalTrafficPolicy: {{ .externalTrafficPolicy }}
  {{- end }}
```

---

done